libs/core/core/hooks/use-device-detached-effect.ts

```diff
@@ -26,13 +28,18 @@ export const useDeviceDetachedEffect = () => {
   const downloadProcessing = useSelector(
     ({ update }: ReduxRootState) => update.downloadState
   )
-  const handleActiveDeviceDetached = useHandleActiveDeviceDetached()
+  const devices = useSelector(getDevicesSelector)
+  const redirectOnActiveDeviceDetached = useRedirectOnActiveDeviceDetached()
 
   useEffect(() => {
     return registerDeviceDetachedListener(
       async (properties: DeviceBaseProperties) => {
         dispatch(removeDevice(properties))
 
+        if (devices.length < 2) {
+          dispatch(setSelectDeviceDrawerOpen(false))
+        }
```

Comment:
I've noticed that when executing method 'redirectOnActiveDeviceDetached()' there are similar conditions inside it, won't we do the same things?

Reply:
after sync https://github.com/mudita/mudita-center/pull/1687/commits/9183a4c36a55af25906ec1da794195de795ae77e